docker/dockerfiles/Dockerfile

```diff
@@ -105,23 +105,8 @@ WORKDIR /tmp
 RUN locale-gen en_US.UTF-8
 
 # Install MySQL.
-#
-# We explicitly target MySQL 5.7, and because of that we also explicitly target
-# Ubuntu Bionic (18.x).
-#
-# We also explicitly install both mysql-client and mysql-server, so that we can
-# also explicitly define the versions of both; normally only the latter is
-# required since it depends on the former.
-#
-# Once we update to MySQL 8+, this can be simplified.
-RUN curl -sSLOJ https://dev.mysql.com/get/mysql-apt-config_0.8.22-1_all.deb && \
-    echo "mysql-apt-config mysql-apt-config/select-server select mysql-5.7" | /usr/bin/debconf-set-selections && \
-    echo "mysql-apt-config mysql-apt-config/repo-codename select bionic" | /usr/bin/debconf-set-selections && \
-    dpkg -i mysql-apt-config_0.8.22-1_all.deb || apt-get --fix-broken --yes install && \
-    rm -rf mysql-apt-config_0.8.22-1_all.deb && \
-    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B7B3B788A8D3785C && \
-    apt-get update && \
-    apt-get install --yes libmysqlclient-dev mysql-client=5.7.42-1ubuntu18.04 mysql-server=5.7.42-1ubuntu18.04
+RUN apt-get update && \
+    apt-get install --yes libmysqlclient-dev mysql-client mysql-server
 
 # disable auth for local dashboard mysql db
 RUN service mysql start && \
```

Comment:
Does it matter that this `service mysql start` comes before the mysql homedir changes below?

Reply:
I don't think so; I don't remember exactly where we encountered the homedir error, but it doesn't prevent the commands in this block from executing successfully, and because we stop the service again afterwards this block shouldn't disrupt the homedir changes from being applied successfully. It seems likely to me that they would also work in the other order, but because they're working as-is, I'm inclined to leave it.